src/dmd/dmodule.d

```diff
@@ -284,6 +284,27 @@ extern (C++) class Package : ScopeDsymbol
         }
         return null;
     }
+
+    /**
+     * Checks for the existence of a package.d to set isPkgMod appropriately
+     * if isPkgMod == PKGunknown
+     */
+    final void resolvePKGunknown()
+    {
+        if (isPkgMod == PKG.unknown)
```

Comment:
My dream: that one day all functions ending with a single long `if` are written to first eliminate the negation of that `if` as the uninteresting case, to then proceed with the interesting part without indentation.